### PR TITLE
feat: adding cronjob option for collector

### PR DESCRIPTION
**Pull request by sleepyStick**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind fix

/kind feat
> /kind build
> /kind chore
> /kind ci
> /kind docs
> /kind style
> /kind refactor
> /kind perf
> /kind test

**What this PR does / Why we need it**:

**Which issue(s) this PR fixes**:

Closes #

**Special notes for your reviewer**:
note that users will need to edit the env var values

### CODE REVIEW

**Comment by sleepyStick** [comment]
@googlebot I signed it!

**Comment by hongalex** [comment]
Some general comments:
The commit message should be an imperative statement (use "add" instead of "adding"). For more info, see [this StackOverflow page](https://stackoverflow.com/questions/3580013/should-i-use-past-or-present-tense-in-git-commit-messages)

It would be good to include a section on what this does in the deploying to GKE page and how to deploy (`kubectl apply`).

/gcbrun

File: deploy/gke/collector.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: collector
+  namespace: open-saves-namespace
+spec: 
+  schedule: "10 0 * * *"
```
Comment (hongalex) [inline deploy/gke/collector.yaml:7]: If I'm reading this properly, this reads as "run on 12:10am" every day. Is this your intent?

**Comment by yuryu** [comment]
Moved to #286

**Pull request closed without merging**